### PR TITLE
Add custom footer slot for Modal component

### PR DESCRIPTION
**Pull request by pchaudh18**
## Description

References #1728

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by msankaran0712 — CHANGES_REQUESTED** [review]

**Review by msankaran0712 — CHANGES_REQUESTED** [review]
Please use the attachment files for the latest code

File: stencil-workspace/src/components/modus-modal/modus-modal.spec.ts [hunk 1]
```diff
@@ -30,7 +30,9 @@ describe('modus-modal', () => {
                 <div class="body">
                   <slot></slot>
                 </div>
-                <div class="footer"></div>
+                <div class="footer">
```
Comment (coliff) [inline stencil-workspace/src/components/modus-modal/modus-modal.spec.ts:33]: Can this be `<footer class="footer">` instead of a div?

**Comment by pchaudh18** [comment]
Hi
I have updated the branch with new changes for the modus-modal and some auto-updated files

**Review by msankaran0712 — APPROVED** [review]

File: stencil-workspace/src/components/modus-modal/modus-modal.spec.ts (hunk 1 above)
Comment (msankaran0712) [reply stencil-workspace/src/components/modus-modal/modus-modal.spec.ts:33]: yes we can do that. @palakchaudhary18 Can you please replace the divs with header and footer HTML elements and remove the css classes 'header' and ''footer' too.

Expected:
image
 
image

Comment (pchaudh18) [reply stencil-workspace/src/components/modus-modal/modus-modal.spec.ts]: updated the code

**Review by msankaran0712 — APPROVED** [review]

**Review by coliff — APPROVED** [review]

**Pull request merged**